src/common/transformations/src/transformations/common_optimizations/gelu_fusion.cpp

```diff
@@ -318,18 +325,26 @@ ov::pass::GeluFusionWithTanh::GeluFusionWithTanh() {
         auto gelu = std::make_shared<ov::op::v7::Gelu>(x_output, op::GeluApproximationMode::TANH);
 
         gelu->set_friendly_name(m.get_match_root()->get_friendly_name());
+
         ov::copy_runtime_info(
             {
                 pattern_to_output.at(pow).get_node_shared_ptr(),
                 pattern_to_output.at(mul_0).get_node_shared_ptr(),
                 pattern_to_output.at(mul_1).get_node_shared_ptr(),
-                pattern_to_output.at(mul_2).get_node_shared_ptr(),
-                pattern_to_output.at(mul_3).get_node_shared_ptr(),
                 pattern_to_output.at(tanh).get_node_shared_ptr(),
                 pattern_to_output.at(add_0).get_node_shared_ptr(),
                 pattern_to_output.at(add_1).get_node_shared_ptr(),
             },
             gelu);
+        if (pattern_to_output.find(mul_2_1) != pattern_to_output.end())
```

Comment:
can we simplify it and the code above?
e.g.
```
vector<Node> pattern_nodes = {pow, mul_0, mul_1, tanh, add_0, add_1, mul_2_1, mul_2_2, mul_3_1, mul_3_2};
vector<Node> cp_rt_info_nodes;
for (const auto& pattern_node : pattern_nodes) {
     if (pattern_to_output.count(pattern_node.get_node_shared_ptr())) {
         cp_rt_info_nodes.push_back(pattern_node);
    }
}
ov::copy_runtime_info(cp_rt_info_nodes, gelu);
```
or something like that?

Reply:
thanx for the review! applied.